src/Microsoft.DotNet.Build.Tasks/Targets/versioning.targets

```diff
@@ -0,0 +1,43 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="GetDailyBuildNumber" 
```

Comment:
Now that I see its usage I don't think this would be the same file that would generate the AssemblyVersion.cs so I think we should probably rename the file to something more specific to its task. Perhaps UpdateBuildNumber.targets  or something similar.